commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -8,55 +8,84 @@ import squants.motion.AngularVelocity
 import squants.{Acceleration, Dimensionless, Length, Percent, Velocity}
 import GenericValue._
 
+trait UnicycleProperties {
+  val maxForwardVelocity: Velocity
+  val maxTurnVelocity: AngularVelocity
+
+  val forwardControlGains: PIDFProperUnitsConfig[Velocity, Acceleration, Length, Dimensionless]
```

Comment:
So I took a look at this, and I don't really see the point of it. Could you please explain why you think this is necessary?